_lessons/python-scatter/04b-openmp.md

```diff
@@ -19,9 +19,9 @@ cd openmp
 
 ## Why implement OpenMP parallelisation
 
-One way to speed up your application is using the available resources more efficiently. This approach was used while porting our Python code to C++ by removing the interpreter's overhead. Here we will improve performance by using more resources instead.
+One way to speed up your application is using the available resources more efficiently. This approach was used while porting our Python code to C/C++ by removing the interpreter's overhead. Here we will improve performance by using more resources instead.
 
-Most modern computers have multi-core CPUs and we can use two or more of these cores to execute instructions in parallel. All cores can access the same, shared memory.
+Most modern computers have multi-core CPUs and we can use two or more of these cores to execute instructions in parallel. All cores in one CPU can access the same, shared memory.
```

Comment:
All cores of a CPU can access the same, shared memory. Furthermore, all the CPUs belonging to a node also share the same memory. Given that supercomputers often have tens of cores per node, anywhere between 20-80 instructions can in principle be executed in parallel.